lib/components/src/controls/Object.tsx

```diff
@@ -244,7 +244,7 @@ const getCustomStyleFunction: (theme: Theme) => JsonTreeProps['getStyle'] = (the
 });
 
 export const ObjectControl: FC<ObjectProps> = ({ name, value, onChange }) => {
-  const theme = useTheme<Theme>();
```

Comment:
This shows a small, subtle, breaking change. I'm open to reverting this. 

Previously you had to provide a type param here. Now you don't have to - so it can be just skipped. However, this means that the previously valid code is no longer valid:
```ts
const theme = useTheme<Theme>();
```
We could adjust the signature of `useTheme` to look something like that:
```ts
declare function useTheme<TTheme = Theme>(): TTheme;
```
This way you wouldn't have to provide a generic, but also you could provide it. While you shouldn't be parametrizing it with just "any" Theme object... this would avoid that small breaking change.

Reply:
```ts
declare function useTheme<TTheme = Theme>(): TTheme;
```
Sounds like a good idea!

Reply:
I've added this change, the whole PR should be ready to merge now :)